tests/tests/test-author-mapping.ts

```diff
@@ -11,6 +11,7 @@ import {
   RANDOM_PRIV_KEY,
 } from "../util/constants";
 import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import { createBlockWithExtrinsic } from "../util/substrate-rpc";
```

Comment:
Why not improve it and simply use the `createBlock` which supports including transactions?
It would be more consistent with the rest and it also allows multiple transactions in 1 block

Reply:
Because createBlock doesn't return the events. Are you saying I should modify createBlock to return the events?

Reply:
It think it would have been better to have only 1 way to create block with transactions, but I suppose we can refactor it another time. There are other parts to refactor too